doc/docs/reference/core_lookuplists.md

```diff
@@ -16,3 +16,26 @@ for l in lookuplists.lookuplists():
 
 # -> Drug, Gender, Condition et cetera
 ```
+
+
+## {{ LookupListCls }}.objects.search(some_str, case_sensitive=False, contains=False)
+A manager method that takes a string and will search accross the lookup list itself or synonyms. If `contains=True` it will look for partial matches. By default it is case
+insensitive, to change this behaviour query with `case_sensitive=True`
+
+```python
+from opal.core import Antimicrobial
+
+Antimicrobial.objects.search("Difficulty Swallowing")
+## returns the Antimicrobial Dysphagia
+```
+
+## {{ LookupListCls }}.objects.search_many(list_of_strs, case_sensitive=False, contains=False)
+Does the same as the above put takes a list of strings
```

Comment:
Yeah, but I want to link to this section by `/blah/#this` and get a description of this !
(Though by all means don't do this now if we're going to think about general API changes)